src/transformers/modeling_utils.py

```diff
@@ -808,7 +809,13 @@ def _load_state_dict_into_meta_model(
     for old_key, new_key in zip(old_keys, new_keys):
         state_dict[new_key] = state_dict.pop(old_key)
 
-    for param_name, param in state_dict.items():
+    # Show shard-level progress. Useful to monitor quantization progress
+    quant_show_progress = False
+    if hf_quantizer is not None:
+        if hasattr(hf_quantizer, "show_progress"):
+            quant_show_progress = hf_quantizer.show_progress
```

Comment:
Maybe we can set this as a property of the `HfQuantizer` class cc @younesbelkada